testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveCompression.java

```diff
@@ -83,6 +83,34 @@ public void testSnappyCompressedParquetTableCreatedInHive()
         onHive().executeQuery("DROP TABLE " + tableName);
     }
 
+    @Test(groups = HIVE_COMPRESSION)
+    public void testSnappyCompressedLargeCreateTableAsInTrino()
+    {
+        runSnappyCompressedLargeCreateTableAsInTrino(false);
+    }
+
+    @Test(groups = HIVE_COMPRESSION)
+    public void testSnappyCompressedLargeCreateTableAsInTrinoWithNativeWriter()
```

Comment:
For the https://github.com/trinodb/trino/issues/5518 bug, is CTAS relevant? or any large write was affected? (assuming the latter, so might be worth reconsidering the name)